Expecto.FsCheck/FsCheck.fs

```diff
@@ -71,7 +71,7 @@ module ExpectoFsCheck =
                           (String.concat " " data.Labels)
 
               let focus =
-                sprintf "Focus on error:\n\t%s (%A, %A) \"%s\"" methodName (uint64 std) (uint64 gen) name
+                sprintf "Focus on error:\n\t%s (%A, %A, 0) \"%s\"" methodName (uint64 std) (uint64 gen) name
```

Comment:
What is the motivation for always printing zero here?

Reply:
This is so the user can copy and paste. The FsCheck 2 module will just ignore the third value